src/appchooserdialog.cpp

```diff
@@ -253,6 +253,9 @@ void AppChooserDialog::accept() {
 }
 
 void AppChooserDialog::onSelectionChanged() {
+    // if we are not in app menu view then don't change the state of OK button
+    if(ui->tabWidget->currentIndex() != 0)
+        return;
```

Comment:
Thanks for finding and fixing the problem!

The root cause of the problem is `AppMenuView::onMenuCacheReload`: menu-cache may be reloaded while the tab has been switched.


To follow the existing code style, please usse curly brackets and put the comment inside it. Please also change the code comment to something like this:

```c++
    if(ui->tabWidget->currentIndex() != 0) {
        // the election may be reset by menu-cache,
        // while the app menu view is not shown
        return;
    }

```